packages/components/src/types/common.ts

```diff
@@ -0,0 +1,13 @@
+export type TextWithA11y = {
```

Comment:
Thoughts on an `index.ts` for the types folder? Making them all available as a group instead of per file basis.

Reply:
Added `index.ts` file